astrometry/include/PixelMap.h

```diff
@@ -332,5 +334,37 @@ namespace astrometry {
     void operator=(const ColorTerm& rhs) = delete;
   
   };
+
+  class ASTMap: public PixelMap {
+    // Class representing a constant shift of position
+  public:
+    ASTMap(const ast::Mapping& mapping_, std::string name_=""):
+      PixelMap(name_), mapping(mapping_) {}
+
+    virtual PixelMap* duplicate() const;
+
+    ~ASTMap() = default;
+    
+    static string type() {return "AST";}
+
+    ast::Mapping mapping;
+
+    void toWorld(double xpix, double ypix,
+                 double &xworld, double &yworld,
+                 double color=astrometry::NODATA) const;
+    void toPix(double xworld, double yworld,
+               double &xpix, double &ypix,
+               double color = astrometry::NODATA) const;
+
+#ifdef USE_YAML
```

Comment:
Do you know if `USE_YAML` is actually set in our builds?  If not, it might be cleaner to leaf this block out entirely.

Reply:
Yes, yaml is built into the mappings and if USE_YAML is false some key classes don't get built. Long term, it would be nice to remove this dependence though.